worker-sandbox/tests/d1.rs

```diff
@@ -0,0 +1,115 @@
+use worker_sandbox::{CreatePerson, Person};
+
+use crate::util::{expect_wrangler, get, post, delete};
+
+mod util;
+
+fn drop_table() {
+    let response = post("d1/exec", |rb| rb.body("drop table if exists people;"));
+
+    assert!(response.status().is_success());
+}
+
+fn create_table() {
+    let query = "CREATE TABLE IF NOT EXISTS people ( \
+            id INTEGER PRIMARY KEY, \
+            name TEXT NOT NULL, \
+            age INTEGER NOT NULL)";
+    let response = post("d1/exec", |rb| rb.body(query));
+    assert!(response.status().is_success());
+}
+
+fn setup() {
+    drop_table();
+    create_table();
+}
+
+#[test]
+fn d1_create_table() {
+    expect_wrangler();
+    setup();
+}
+
+#[test]
+fn d1_insert_prepare_first() {
+    expect_wrangler();
+    setup();
+    let person = create_person();
+
+    assert_eq!("Ada LoveLace", person.name);
+    assert_eq!(32, person.age);
+}
+
+#[test]
+fn d1_select_prepare_all() {
```

Comment:
I tried to highlight the API that is being used in the tests, but I think they might need to be a bit more verbose